packages/gatsby-remark-images/src/gatsby-browser.js

```diff
@@ -0,0 +1,26 @@
+const { imageClass, imageBackgroundClass, imageWrapperClass } = require(`./classes`)
+
+exports.onRouteUpdate = () => {    
+    const imageWrappers = document.querySelectorAll(`.${imageWrapperClass}`)
+
+    Array.prototype.forEach.call(imageWrappers, (imageWrapper) => {
```

Comment:
I think it would be much more readable if you would use:

```
const imageWrappers = Array.from(document.querySelectorAll(`.${imageWrapperClass}`))

imageWrappers.forEach(imageWrapper => {
```

Reply:
I remember why I didn't do this. There is no polyfill currently loaded and felt probably wasn't worthwhile in this case since readability was similar. Do you think it's worth adding a polyfill or keep as is @pieh?